CMakeLists.txt

```diff
@@ -89,9 +89,15 @@ else ()
   #   interfaces, etc.
   # disable -Wpointer-arith: this is a GCC extension, and doesn't work in MSVC.
   add_definitions(
-    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g
+    -Wall -Wextra -Werror -Wno-unused-parameter -Wpointer-arith -g -std=c++11
```

Comment:
Guys, you realize this was a drive-by-troll, right? You've made it to Horse wasm, so I guess that's nice.